index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wasyihun-Portfilo</title>
+    <link
+      href="https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="CSS/style.css" />
+  </head>
+  <body>
+    <header class="navbar">
+      <h3 class="M-logo">Mylogo</h3>
+      <div class="nav_toggle" id="nav-toggle">
+        <i class="bx bx-menu"></i>
+      </div>
```

Comment:
- [ ] Good job adding a menu button also called a hamburger button :clap: , but you should wrap it in a `nav` tag to meet the requirement. Kindly fix that